charts/tekton-apps/templates/wordpress/argocd_applications.yaml

```diff
@@ -22,161 +17,171 @@ metadata:
     argocd.argoproj.io/sync-wave: {{ $argocd.syncWave | default "210" | quote }}
 spec:
   destination:
-    namespace: {{ $project.argocd.namespace }}
+    namespace: {{ $project.argocd.namespace | required ".argocd.namespace is required in project settings"}}
     server: https://kubernetes.default.svc
   ignoreDifferences:
-  - group: v1
-    jsonPointers:
-    - /metadata/labels
-    kind: secret
-  {{ if (ternary $argocd.ignoreDeploymentReplicasDiff false (hasKey $argocd "ignoreDeploymentReplicasDiff")) }}
-  - group: apps
-    kind: Deployment
-    jsonPointers:
-    - /spec/replicas
-  {{- end }}
+    - group: v1
+      kind: secret
+      jsonPointers:
+        - /metadata/labels
+    {{- if $argocd.ignoreDeploymentReplicasDiff }}
+    - group: apps
+      kind: Deployment
+      jsonPointers:
+        - /spec/replicas
+    {{- end }}
   project: {{ $project.project }}
   source:
     chart: wordpress
     helm:
       values: |
-        wordpressSkipInstall: false
         image:
           repository: bitnami/wordpress
-          tag: {{ $component.wordpress.imageTag | default "5.7.2" }}
-          debug: true
+          tag: {{ (.image).tag | default "5.7.2" }}
```

Comment:
latest is `6.1.1` supported by bitnami - let's use it.
```
➜ helm search repo wordpress                               
NAME                              	CHART VERSION	APP VERSION	DESCRIPTION                                       
bitnami/wordpress                 	15.2.40      	6.1.1      	WordPress is the world's most popular blogging ...
bitnami/wordpress-intel           	2.1.31       	6.1.1      	DEPRECATED WordPress for Intel is the most popu...
saritasa/saritasa-tekton-apps     	0.2.0        	v0.28.2    	A Helm chart for tekton apps (rbac, eventlisten...
saritasa/saritasa-tekton-pipelines	0.1.30       	           	A Helm chart for Tekton Pipelines  Implements: ...
```

Reply:
Done